prepackPy/na_counter.py

```diff
@@ -4,20 +4,70 @@
 import warnings
```

Comment:
do we need to import this package?

Reply:
Nice catch.   Addressed in [04d81b9](https://github.com/UBC-MDS/prepackPy/pull/21/commits/04d81b924e237c18d6c3061dcf90f7e446592096)